blog/docs/articles/Building-Stateful-applications-with-Knative-and-Restate.md

```diff
@@ -0,0 +1,249 @@
+# Building Stateful applications with Knative and Restate
+
+Knative revolutionized developing and operating serverless applications on Kubernetes, but it is still quite challenging to build stateful applications on top of it.
+
```

Comment:
Usually there is an author tag at the top https://github.com/knative/docs/blob/main/blog/docs/articles/llm-agents-overview.md

> **Author: [Calum Murray](https://www.linkedin.com/in/calum-ra-murray/), Software Engineering Intern @ Red Hat**

do you want to add yours, so that people know who wrote it, in case they want to reach out?

Reply:
Can do this, thanks!

Reply:
I added myself and my colleague @gvdongen which helped me :)